homework_1/zad2/taxCalculation/taxes.go

```diff
@@ -0,0 +1,18 @@
+package taxCalculation
+
+type taxLevel struct {
+	UpperBound float64
+	Percentage float64
+}
```

Comment:
I see you're using a mix of uppercase and lowercase naming. Not sure if this was intentional or not, but it does matter in Go - names starting with an uppercase letter are public (accessible by anyone), while names starting with lowercase letters are unexported (accessible only within the same package).
For this homework assignment it's not so important, but in general it does matter how you name your structs/functions/etc.